### DISCUSSION TITLE
File to import not found or unreadable

### DISCUSSION THREAD

**Discussion by Sweetyy** [quasarframework/quasar#10664] [General - Components / Directives / etc] [answered]
Hi everyone!

### I really need your help for this error when I run the project with Quasar 2.0.4 :

![quasar](https://user-images.githubusercontent.com/15823138/133280062-7b9d389f-baa3-4e7d-8438-df326c5597f8.png)

### Here is my files structure for CSS :

![quasar2](https://user-images.githubusercontent.com/15823138/133280346-eb37e915-a234-4fb3-a680-f12f40103dde.png)

### Finally my quasar.variables.scss file :

![quasar3](https://user-images.githubusercontent.com/15823138/133280503-222d3484-5784-4212-8991-e7320c317d61.png)

I can't find where is the problem and really need your help to figure it out plz.

Thanks in advance!

**Comment by MilosPaunovic** [comment]
This doesn't seem to be related with the `quasar.variables.scss` file, but with the one used by Quasar, internally. First idea is for you to delete `.quasar` and `node_modules` folders and run a fresh `npm i`.

**Reply by Sweetyy** [reply]
I already tried that but with `yarn install.` By the way I'm using docker with this project. Do you think it could be related to the docker image persistent ?

**Comment by Sweetyy** [comment, marked as answer]
I found what was the problem.

It was in the package.json file concerning this package : `"sass-loader": "^6.0.6",`

I removed it and now it works like a charm !
